src/components/Counter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import styles from "./Counter.module.scss";
+
+interface CounterState {
+	value: number;
+}
+
+export default class Counter extends Component<{}, CounterState> {
+	state: CounterState = {
+		value: 0,
+	};
+
+	decrementValue = (): void => {
+		this.changeValue(-1);
+	};
+
+	incrementValue = (): void => {
+		this.changeValue(1);
+	};
+
+	changeValue = (diff: number): void => {
+		this.setState((state) => ({
+			value: state.value + diff,
+		}));
+	};
+
+	render() {
+		return React.createElement(
+			"div",
+			{
+				className: styles.counter,
+			},
+			[
+				React.createElement(
```

Comment:
it is great to understand how react without JSX works. But I know no real need to use this. It is better to create custom Button component with all styles and maybe some custom logic inside.